services.go

```diff
@@ -95,6 +96,10 @@ func (s *serviceSet) dispatch(ctx context.Context, serviceName, methodName strin
 		return nil, err
 	}
 
+	if isNil(resp) {
+		return nil, errors.New("ttrpc: error unsupported response nil payload with nil error")
```

Comment:
I think `ttrpc: marshal called with nil` is fine. just my two cents :P

Reply:
Hi @fuweid,  updated.